src/mmw/mmw/settings/layer_settings.py

```diff
@@ -89,6 +89,9 @@
         'url': 'https://s3.amazonaws.com/com.azavea.datahub.tms/'
                'nlcd/{z}/{x}/{y}.png',
         'overlay': True,
+        'maxNativeZoom': 13,
+        'maxZoom': 18,
+        'opacity': 0.618,
```

Comment:
Consider documenting why this number is so specific or selecting a less precise number (0.6, 0.75, etc)